posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -118,7 +118,7 @@ Note: your browser might show a security prompt that you'll need to accept due t
 
 image::img/blog/ui_login.png[ui login ,width=70%,align="center"]
 
-Stay tuned for a blog specifically regarding Admin Center which will showcase the various tools and functionality it provides.
+For a deeper dive into Admin Center and its various tools and functionality it provides, visit the link:{url-prefix}/blog/2021/04/21/admin-center-21004.html[Admin Center: a GUI for Open Liberty] blog post.
```

Comment:
For a deeper dive into Admin Center and its various tools and functionality it provides
--->
For a deeper dive into Admin Center and the various tools and functionality it provides

Reply:
Updated